Feature profile page skeleton

m3tron opened the pull request:
Refactored dashboard with AppBar and Drawer which included friendsList.

m3tron committed:
added toolbar

m3tron committed:
dev merge

m3tron committed:
Merge branch 'dev' of github.com:hatchways/team-pretzel into feature-profile-page-skeleton

m3tron committed:
pre-merge commit

m3tron committed:
post dev merge

m3tron committed:
Merge branch 'dev' of github.com:hatchways/team-pretzel into feature-profile-page-skeleton

m3tron committed:
refactored dashboard with navbar & friendDrawer

m3tron committed:
added key to avatar

m3tron committed:
moved key to each list item

m3tron committed:
redirect to login after clicking logout

m3tron committed:
removed async keyword from getme

skassam21 commented on client/src/App.jsx:
```diff
@@ -5,7 +5,7 @@ import { MuiThemeProvider, CssBaseline } from "@material-ui/core";
 import { theme } from "./themes/theme";
 import AuthPage from "./pages/AuthPage";
-import Dashboard from "./pages/Dashboard";
+import Dashboard from "./pages/Dashboard/index";
```
let's be consistent with naming conventions - I more commonly see with React the name of the files are the name of the components and are PascalCase

skassam21 commented on client/src/components/AppBarDrawer/index.js:
```diff
@@ -0,0 +1,179 @@
+  // content: {
+  //   flexGrow: 1,
+  //   padding: theme.spacing(3)
+  // },
```
remove commented out code

skassam21 commented on client/src/components/AppBarDrawer/index.js:
```diff
@@ -0,0 +1,179 @@
+
+  return (
+    <div className={classes.root}>
+      <CssBaseline />
```
I beleive CssBaseline shouold only exist at the top level react component

skassam21 commented on client/src/components/FriendList/index.js:
```diff
@@ -0,0 +1,25 @@
+import Friends from "../Friends";
+
+const useStyles = makeStyles(theme => ({
+  //title: { flexGrow: 1 }
```
again commented out code

skassam21 commented on client/src/components/FriendList/index.js:
```diff
@@ -0,0 +1,25 @@
+
+  return (
+    <>
+      <Typography variant="h4" className={classes.title}>
```
need some padding or margin here for the friends title

skassam21 reviewed (commented):
Small comments

m3tron committed:
Resolved conversations

- Changed file names to match directory structure
- Removed commented code
- Added padding to friends title
- Used hooks to fetch user and friends

m3tron committed:
added skipping effects to friends and user

m3tron committed:
changes name on navbar on submit of name update

m3tron committed:
Merge branch 'dev' of github.com:hatchways/team-pretzel into feature-profile-page-skeleton

m3tron committed:
added poll dialog to create poll button

k-awe-some commented on client/src/App.jsx:
```diff
@@ -5,7 +5,7 @@ import { MuiThemeProvider, CssBaseline } from "@material-ui/core";
 import { theme } from "./themes/theme";
 import AuthPage from "./pages/AuthPage";
-import Dashboard from "./pages/Dashboard";
+import Dashboard from "./pages/Dashboard.js";
```
can we possibly combine the two Dashboard components into one `Dashboard.jsx` file?

k-awe-some reviewed (changes requested):
Profile page looks solid yet Create new poll is not letting user upload images, code looks fine to me so I'm thinking conflicts come from `PollDialog` being used in 2 places (`AppBarDrawer` and `NavBar`) ?!

also if we could all use `.jsx` for React components that'd be more consistent.

m3tron committed:
Applied requested changes

- Changed file type to .jsx
- Removed Navbar and Dashboard because changes were already merged into AppBarDrawer and Dashboard.js
